frontend/language/src/en.json

```diff
@@ -25,7 +25,9 @@
   "header_form.factory.form.builder.link": "Form Builder",
   "header_form.factory.home.link": "Home",
   "header_form.factory.title.link": "Form Factory",
+  "login_page.auth.error": "You must be logged in to access the requested page.",
   "login_page.login.button": "Log in",
+  "login_page.login.error": "The server could not process your request. Please double check your credentials and try again.",
```

Comment:
What about this "Could not sign you in, Please ensure your username and password are correct and try again."